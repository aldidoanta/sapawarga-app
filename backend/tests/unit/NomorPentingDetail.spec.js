import { shallowMount, createLocalVue } from "@vue/test-utils"
import Show from '@/views/nomorPenting/show'
// import PhotoBox from "@/components/PhotoBox"
import ElementUI from "element-ui";
const sampleImage = require("@/assets/user.png")

const localVue = createLocalVue()
localVue.use(ElementUI)

const $route = {
  // http://localhost:9527/backend/#/nomor-penting/show/140
  path: '/nomor-penting/show/140',
  params: { id: '578' },
}

const factory = (values = {}) => {
  return shallowMount(Show, {
    // data: { ...values },
    propsData: { ...values },
    localVue,
    mocks: {
      $route
    }
  })
}

describe('Nomor Penting Show Detail', () => {
  it('renders main component', () => {
    const wrapper = factory()

    expect(wrapper.html()).toMatchSnapshot()
  })
})
