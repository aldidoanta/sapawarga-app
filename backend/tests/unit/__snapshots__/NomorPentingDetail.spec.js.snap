// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nomor Penting Show Detail renders main component 1`] = `
<div class="app-container">
  <el-row-stub tag="div" gutter="10" justify="start" align="top">
    <el-col-stub span="24" tag="div" xs="24" sm="24" md="24" lg="7" xl="5" class="col-left">
      <photobox-stub height="220px" class="image"></photobox-stub>
      <vue-friendly-iframe-stub id="map" src="https://maps.google.com/maps?q=null,null&amp;hl=es;z=14&amp;output=embed" style="border: 0px; display: none;"></vue-friendly-iframe-stub>
    </el-col-stub>
    <el-col-stub span="24" tag="div" xs="23" sm="23" md="23" lg="16" xl="18" class="col-right">
      <el-card-stub>
        <div class="clearfix"><span>Nomor Telepon</span></div>
        <el-table-stub data="" fit="true" selectonindeterminate="true" indent="16" style="width: 100%;">
          <el-table-column-stub type="default" prop="title" width="180" resizable="true" filtermultiple="true" sortorders="ascending,descending,"></el-table-column-stub>
          <el-table-column-stub type="default" prop="content" resizable="true" filtermultiple="true" sortorders="ascending,descending,"></el-table-column-stub>
        </el-table-stub>
      </el-card-stub>
    </el-col-stub>
  </el-row-stub>
</div>
`;
